templates/extra/privacy.html

```diff
@@ -41,7 +41,7 @@ <h1 class="privacy-title">Privacy</h1>
         <p>Only persons age 18 or older have permission to access our Service. Our Service does not address anyone under the age of 13 ("Children").</p>
         <p>We do not knowingly collect personally identifiable information from children under 13. If you are a parent or guardian and you learn that your Children have provided us with Personal Information, please contact us. If we become aware that we have collected Personal Information from a children under age 13 without verification of parental consent, we take steps to remove that information from our servers.</p>
         <h5>Changes To This Privacy Policy</h5>
-        <p>This Privacy Policy is effective as of September 17, 2015 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
+        <p>This Privacy Policy is effective as of July 11, 2016 and will remain in effect except with respect to any changes in its provisions in the future, which will be in effect immediately after being posted on this page.</p>
```

Comment:
I hadn't read this line before, but actually I think we want to leave this line as it was, as the date referred to when it originally went into effect.